workos/resources/authentication_response.py

```diff
@@ -1,5 +1,5 @@
 from workos.resources.base import WorkOSBaseResource
-from workos.resources.users import WorkOSUser
+from workos.resources.user_management import WorkOSUser
 
 
 class WorkOSAuthenticationResponse(WorkOSBaseResource):
```

Comment:
nice catch on the renaming.

could you move this and the `WorkOSPasswordChallengeResponse` into the `workos.resources.usermanagement` module? (can be in a followup)

Reply:
good catch - I moved those 👍